packages/cli/src/config/schema.ts

```diff
@@ -1012,6 +1016,53 @@ export const schema = {
 		},
 	},
 
+	eventBus: {
+		enabled: {
+			doc: 'Whether event logging is enabled.',
+			format: Boolean,
+			default: true,
+			env: 'N8N_EVENTBUS_ENABLED',
+		},
+		checkUnsentInterval: {
+			doc: 'How often (in ms) to check for unsent event messages. Can in rare cases cause a message to be sent twice. 0=disabled',
+			format: Number,
+			default: 0,
+			env: 'N8N_EVENTBUS_CHECKUNSENTINTERVAL',
+		},
+		destinations: {
+			loadAtStart: {
+				doc: 'Whether destinations are loaded from Db and initialized at startup.',
+				format: Boolean,
+				default: true,
+				env: 'N8N_EVENTBUS_DESTINATIONS_STARTUP',
+			},
+		},
+		logWriter: {
+			syncFileAccess: {
+				doc: 'Whether all file access happens synchronously within the thread.',
+				format: Boolean,
+				default: false,
+				env: 'N8N_EVENTBUS_LOGWRITER_SYNCFILEACCESS',
+			},
+			keepLogCount: {
+				doc: 'How many event log files to keep.',
+				format: Number,
+				default: 3,
+				env: 'N8N_EVENTBUS_LOGWRITER_KEEPLOGCOUNT',
+			},
+			maxFileSizeInKB: {
+				doc: 'Maximum size of an event log file before a new one is started.',
+				format: Number,
+				default: 102400,
```

Comment:
```suggestion
				default: 102400, // 100kb
```

Reply:
it's in KB, so this is 100MB. but still makes sense to add a comment to avoid this confusion :)